src/OptiHPLCHandler/empower_handler.py

```diff
@@ -244,6 +244,14 @@ def GetSystemNames(self, node: str) -> List[str]:
         response = self.connection.get(endpoint=endpoint)
         return response.json()["results"]
 
+    # SLCB
```

Comment:
There is no reason to keep this, we have git blame to tell us who did what when.